src/styles/blocks/catalog.scss

```diff
@@ -21,5 +21,11 @@
     &:not(:nth-child(4n)) {
       margin-right: 48px;
     }
+
+    &:hover {
+      transition: 300ms;
+      transform: scale(120%);
+      color: #34568b;
+    }
```

Comment:
at the `hover` state you need only set what should be changed
`transition` rule remove to the &__card